async_firebase/messages.py

```diff
@@ -215,3 +215,45 @@ class PushNotification:
 
     message: Message
     validate_only: t.Optional[bool] = field(default=False)
+
+
+class FcmPushMulticastResponse:
+    """The response received from a batch request to the FCM API.
+
+    The interface of this object is compatible with BatchResponse object of
+    the Google's firebase-admin-python package.
+    """
+
+    def __init__(self, responses):
+        self._responses = responses
+        self._success_count = len([resp for resp in responses if resp.success])
+
+    @property
+    def responses(self):
+        """A list of ``messaging.SendResponse`` objects (possibly empty)."""
+        return self._responses
+
+    @property
+    def success_count(self):
+        return self._success_count
+
+    @property
+    def failure_count(self):
+        return len(self.responses) - self.success_count
+
+
+class FcmPushResponse:
```

Comment:
I guess two new classes should be considered `dataclass`. What do you think?

Reply:
Let's think about it later